### DISCUSSION TITLE
packageRules not getting kicked in

### DISCUSSION THREAD

**Discussion by ankitabhopatkar13** [renovatebot/renovate#9933] [Request Help]
I want to integrate renovate for updating the dependencies.

I have a package.json with the following dependency pattern - 

```
{
  "name": "@myorg/myorg-test",
  "version": "2.0.0",
  "scripts": {},
  "devDependencies": {
    "@types/node": "14.0.1",
    "@myorg/some-pkg": "4.31.0"
  }
}

```
I am changing config.js to only match _@myorg_. But I see, renovate is always updating node dependency but never my own package.

_myorg_ is a private registry on Gitlab. I have tried hostRules as well as npmrc approach to authenticate to my registry.

The config.js snippet is as follows - 

```
    automerge: false,
    packageRules: [{
      matchPaths: ['package.json'],
      matchPackagePatterns: ["^@myorg/"],
      groupName: 'myorg packages',
  }],
```

Any help will be appreciated!

**Comment by rarkins** [comment]
It's not possible to guess based on the information so far. Please run with `LOG_LEVEL=debug` in env, and then search the logs for mentions of `@myorg/some-pkg`.

**Reply by ankitabhopatkar13** [reply]
Thanks @rarkins. Setting up the log level made the error clear and I could fix it myself.

**Comment by viceice** [comment]
By default all packages are enabled, so you need to disable packages with a rule if you don't want them updated
